### PR TITLE
add Typora.app

### PR DESCRIPTION
**Pull request by jconley**

### CODE REVIEW

File: Casks/typora.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'typora' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://typora.io/download/typora_latest.zip '
```
Comment (vitorgalvao) [inline Casks/typora.rb]: The space before the `'` should be removed.

Comment (jconley) [inline Casks/typora.rb]: paste fail! fixed.

File: Casks/typora.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'typora' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://typora.io/download/typora_latest.zip'
+  name 'Typora'
+  homepage 'http://typora.io'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/typora.rb]: `:gratis`, it looks like; at least for now.

Comment (jconley) [inline Casks/typora.rb]: done

**Pull request merged**